### PR TITLE
dark mode issue#272 solved 

### LINKED ISSUE #272: add logo & dark/light theme in signup page[Feat]:
### What feature?

i want to add gyanaguru logo in signup page onclick navigate to home page  & dark/light theme option - 
Screenshot (109)

We will see it in the login page just as it is -  

Screenshot (110)

_No response_

### Agreement to Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by mandliyarajendra11**
## Description 📝

The sign-up page did not have a dark mode toggle component, and apart from that, if the website is in dark mode and you go to the sign-up page, some text was not visible due to certain classes.

## Changes 💬

I added the dark mode toggle component and made some changes in the CSS classes, adding some css class related to dark mode . Now, the dark mode option has been successfully added to it .

## Checklist ✅

- [ ] Reviewed the code changes thoroughly.
- [ ] Tested the changes locally and ensured they work as expected.
- [ ] Updated relevant documentation.
- [ ] Added appropriate test cases, if applicable.
- [ ] Considered potential security or performance implications.
- [ ] Code adheres to the project's style guidelines.
- [ ] Addressed all reviewer comments and concerns, if any.

## Screenshots (if applicable)

before when you on dark mode - 
Screenshot (114)

now  - 
Screenshot (115)

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

fixed  #272

### CODE REVIEW

**Comment by mandliyarajendra11** [comment]
@PiyushKalyanpy  check now

File: pages/signup.tsx [hunk 1]
```diff
@@ -64,39 +65,51 @@ const SignUp = () => {
...
-          success();
-          router.push("/dashboard");
-        })
-        .catch((err: any) => {
-          error({ message: err.message });
-        });
+      signup(userData.email, userData.password).then(({ data }: any) => {
+        console.log("aa",data);
+        success();
+        router.push("/dashboard");
```
Comment (lalitmee) [inline pages/signup.tsx:73]: We should first check for the data and then show `success()` and `redirect`. And please remove `console.log()` on line `71`.

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by mandliyarajendra11** [comment]
@lalitmee  done

**Review by PiyushKalyanpy — APPROVED** [review]

**Review by lalitmee — CHANGES_REQUESTED** [review]

File: pages/signup.tsx [hunk 2]
```diff
@@ -44,7 +45,7 @@ const SignUp = () => {
     });
   };
 
-  const error = ({ message }: any) => {
+  const error = ({message}:any) => {
```
[the remark this answers is not in the document]
Comment (mandliyarajendra11) [reply pages/signup.tsx:48]: > Why the type of `message` is `any`? It should be `string` I think.

i haven't updated anything here , i might have just accidentally pressed enter

[the remark this answers is not in the document]
Comment (lalitmee) [reply pages/signup.tsx:48]: I know that, but can you change the type of the `message` to `string`? It's typescript and we use it to put specific types not for `any` for all the things.

**Comment by PiyushKalyanpy** [comment]
Hey @mandliyarajendra11 , we have updated the login and signup screen, if you want to make changes to the new screen please create a new ISSUE.

**Pull request closed without merging**